public/app/features/dashboard/custom_time_ranges/NoTimeRanges.tsx

```diff
@@ -0,0 +1,34 @@
+import React, { PureComponent } from 'react';
+
+export interface Props {
+  add: any;
+}
+
+export default class NoTimeRanges extends PureComponent<Props> {
+  constructor(props) {
+    super(props);
+  }
```

Comment:
You can remove this constructor. `PureComponent` will run this exact code, so a custom constructor is only needed if you do something in it.